.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,22 @@
+# 🐛 Bug Fix
+
+**Related Issue:**
+> Fixes #
+
+### The Bug
+
+<!--
+- What was happening? (e.g., Division by zero caused a segfault)
+- What happens now? (e.g., Returns error 84 gracefully)
+-->
+
+### 🛡️ Robustness & Security
+
+- [ ] This fix prevents a specific crash/vulnerability.
+- [ ] Added a regression test to ensure this doesn't come back.
+
+### ✅ Checklist
+
+- [ ] `make` compiles without error
+- [ ] Tests passed
+- [ ] Error messages are written to error output and exit with 84
```

Comment:
If I fix a logic issue, I don't really care for any error message, nor exiting 84